test/functional/test_ivn.py

```diff
@@ -12,16 +12,21 @@
 import yaml
 import sys
 import shutil
+import time
+import paramiko
 from infrasim import run_command
 from test.fixtures import FakeConfig
 from infrasim.model import CNode
 from infrasim import sshclient
+from infrasim import helper
 from test import fixtures
 
 old_path = os.environ.get('PATH')
 new_path = '{}/bin:{}'.format(os.environ.get('PYTHONPATH'), old_path)
-a_boot_image = os.environ.get('TEST_IMAGE_PATH') or "/home/infrasim/jenkins/data/ubuntu14.04.4.qcow2"
-b_boot_image = "/home/infrasim/jenkins/data/ubuntu14.04.4_b.qcow2"
+a_boot_image = os.environ.get('TEST_IMAGE_PATH') or "/home/infrasim/jenkins/data/cloud-img0.img"
+b_boot_image = os.environ.get('TEST_IMAGE_PATH') or "/home/infrasim/jenkins/data/cloud-img1.img"
```

Comment:
if system environment variable exists, a_boot_image and b_boot_image will point to same file, which is unstable and dangerous.